### PR TITLE
README.md update

### PR DESCRIPTION
**Pull request by apenlor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,55 +93,81 @@ public class Address {
...
+        .body(bodyDslPart)
+        .toPact();
 }
-```
\ No newline at end of file
+```
+
+## Roadmap
+
+Roadmap available under [GitHub Projects section](https://github.com/orgs/sngular/projects/2).
```
Comment (francislainy) [inline README.md:149]: This link does not seem to exist.

Comment (apenlor) [reply README.md:149]: It does exist although it is still private. Pending to make it public.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
+- [Roadmap](#roadmap)
+- [Contributing](#contributing)
+- [License](#license)
+- [Contact](#contact)
 
-### How to configure the POM file
+## Getting Started
+### Requirements
+- JDK +17
+- Having in project a verification library from your choice to have @Max and @Min annotations available (like [Jakarta](https://central.sonatype.com/artifact/jakarta.validation/jakarta.validation-api), [Spring Boot](https://central.sonatype.com/artifact/org.springframework.boot/spring-boot-starter-validation), or similar)
```
Comment (francislainy) [inline README.md]: Having inside your project a verification library of your choice so as to have the `@Max` and `@Min` annotations available, such as Jakarta, Spring Boot or similar.

File: README.md [hunk 3]
```diff
@@ -29,11 +51,31 @@ Just add the dependency on the
...
+Gradle
+
+```groovy
+implementation('com.sngular:pact-annotation-processor:1.0.0')
+```
+
+## Usage
+
+To enable the code generation in a model class, you should annotate it as `@PactDslBodyBuilder`.
+That is the only requirement, all other annotations are optional and used for customize the code generated.
```
Comment (francislainy) [inline README.md]: ..used for customising (ing form)..
The generated code may sound a bit more fluid than code generated.

File: README.md [hunk 4]
```diff
@@ -29,11 +51,31 @@ Just add the dependency on the
...
+|----------------------:|:--------:|:-----:|:--------------------------------------------------------------------------------------------------------------------------------------------|
+| `@PactDslBodyBuilder` |   true   | Class | Main annotation, to be included in classes that want to be processed.                                                                       |
+|            `@Example` |  false   | Field | Used to provide an specific value for a field. If not present in a field, the value created will be random.                                 |
+|                `@Min` |  false   | Field | Defines the maximum value for numeric fields, or number of elements if applied to collections. Will be ignored if an `@Example` is present. |
+|                `@Max` |  false   | Field | Defines the minimum value for numeric fields, or number of elements if applied to collections. Will be ignored if an `@Example` is present. |
+
+`@Example` values are always provided as String. For Dates and ZonedDateTime the only format supported in this version is the one shown in the example.
```
Comment (francislainy) [inline README.md]: which example? Perhaps saying something like "example below"?

File: README.md (hunk 2 above)
Comment (apenlor) [reply README.md]: Applied, thanks!

File: README.md (hunk 3 above)
Comment (apenlor) [reply README.md]: Applied, thanks!

File: README.md (hunk 4 above)
Comment (apenlor) [reply README.md]: Applied, thanks!

**Review by jemacineiras — APPROVED** [review]

**Review by morvader — APPROVED** [review]
LGTM

**Review by jemacineiras — APPROVED** [review]

**Pull request merged**